pentagon/migration/migrations/migration_2_7_1.py

```diff
@@ -0,0 +1,112 @@
+from copy import deepcopy
+
+
+from pentagon import migration
+from pentagon.migration import *
+import yaml
+
+readme = """
+
+# Migration 2.7.1 -> 2.7.2
+
+## This migration:
+- adds kubelet flags that were missing in the last migration to take advantage of the audit policy
+
+
+## Risks:
+- this requires you to roll the cluster
+
+## Follow up tasks:
+- roll the cluster
+
+"""
+
+# Magic to make block formatting in yaml.dump work as expected
+
+
+class folded_unicode(unicode):
+    pass
+
+
+class literal_unicode(unicode):
+    pass
+
+
+def folded_unicode_representer(dumper, data):
+    return dumper.represent_scalar(u'tag:yaml.org,2002:str', data, style='>')
+
+
+def literal_unicode_representer(dumper, data):
+    return dumper.represent_scalar(u'tag:yaml.org,2002:str', data, style='|')
+
+
+yaml.add_representer(folded_unicode, folded_unicode_representer)
+yaml.add_representer(literal_unicode, literal_unicode_representer)
+# https://stackoverflow.com/questions/6432605/any-yaml-libraries-in-python-that-support-dumping-of-long-strings-as-block-liter
+
+audit_settings = {
+    'auditLogPath': '/var/log/kube-apiserver-audit.log',
+    'auditLogMaxAge': 10,
+    'auditLogMaxBackups': 1,
+    'auditLogMaxSize': 100,
+    'auditPolicyFile': '/srv/kubernetes/audit.yaml'
+}
+
+
+class Migration(migration.Migration):
+    _starting_version = '2.7.1'
+    _ending_version = '2.7.2'
+
+    _readme_string = readme
+
+    def run(self):
+
+        old_nodes_file_name = "nodes.yml"
+
+        for item in self.inventory:
+            inventory_path = "inventory/{}".format(item)
+            # If there are no clusters, move on.
+            if not os.path.isdir('{}/clusters/'.format(inventory_path)):
+                continue
+
+            for cluster_item in os.listdir('{}/clusters/'.format(inventory_path)):
+                item_path = '{}/clusters/{}'.format(inventory_path, cluster_item)
+                # Is this a kops cluster?
+
+                # There is a small amount of variation here where some cluster config
+                # directories are `cluster` and some are `cluster-config`
+                # Align these
+                if os.path.isdir("{}/cluster".format(item_path)):
+                    self.move("{}/cluster".format(item_path), "{}/cluster-config".format(item_path))
```

Comment:
Can we add output that we're doing this and why (basically the comment in a logging output to the person running the migration).